docs/content/latest/develop/learn/ttl-data-expiry.md

```diff
@@ -1,30 +1,102 @@
 ---
-title: TTL for data expiry
-linkTitle: TTL for data expiry
+title: 9. TTL for data expiry
+linkTitle: 9. TTL for data expiry
+description: TTL for data expiry
 aliases:
   - /develop/learn/ttl-data-expiry/
+menu:
+  latest:
+    identifier: ttl-data-expiry
+    parent: learn
+    weight: 581
+showAsideToc: true
+isTocNested: true
 ---
+In YCQL there are two types of TTL, the table level TTL and column level TTL. The column TTLs are stored
+with the value of the column. The table level TTL is not stored in DocDB (it is stored
+in master’s syscatalog as part of the table’s schema). If no TTL is present at the column’s value,
+the table TTL acts as the default value.
```

Comment:
table TTL -> table level TTL (everywhere). Already mentioned this in the earlier pass of review comments.